.appveyor.yml

```diff
@@ -0,0 +1,217 @@
+        #    Test the oldest and the latest 32-bit Python versions release. 
+        #    Test with PyQt4 where available, otherwise PyQT5 in each version.
+        #    Test Pyside1 in latest Python version it runs on.
+        #    Test Pyside2 in latest Python version available.
+        #    Test QScintilla on latest Python version available.
+        #    Only test single toolkit installs (e.g. not QT4 and QT5 on same install)
+        
+        # VARIABLES
+        #    NAME: just a comment for distinguishing builds in Appveyor.
+        #    PYQT_VERSION: Determines PyQt versions installation
+        #    PY_QT: Determines Pyside version installation (if present), and is used by 
+        #           Enaml.
+        #    PYTEST_QT_API: In Python 3.4, pytest-qt needs a hint about the right interface.
+        #    QSCINTILLA: Set to "Yes" to install it.
+        
+
+        # 64-bit versions, PyQT, latest to oldest.
+        
+        - NAME: "Py 3.6"
+          PYTHON: "C:\\Python36-x64"
+          PYTHON_VERSION: "3.6.3"
+          PYTHON_ARCH: "64"
+          MINICONDA: C:\Miniconda36-x64
+          PYQT_VERSION: "5"
+
+        - NAME: "Py 3.5"
+          PYTHON: "C:\\Python35-x64"
+          PYTHON_VERSION: "3.5.5"
+          PYTHON_ARCH: "64"
+          MINICONDA: C:\Miniconda35-x64
+          PYQT_VERSION: "5"
+
+# Does not pass tests!
+#        - NAME: "Py 3.4 PyQt 4 v2"
+#          PYTHON: "C:\\Python34-x64"
+#          PYTHON_VERSION: "3.4.8"
+#          PYTHON_ARCH: "64"
+#          MINICONDA: C:\Miniconda34-x64
+#          PYQT_VERSION: "4"
+#          PYTEST_QT_API: "pyqt4v2"
+#
+#        - NAME: "Py 3.4 PyQt 4 v1"
+#          PYTHON: "C:\\Python34-x64"
+#          PYTHON_VERSION: "3.4.8"
+#          PYTHON_ARCH: "64"
+#          MINICONDA: C:\Miniconda34-x64
+#          PYQT_VERSION: "4"
+#          PYTEST_QT_API: "pyqt4"
+
+        - NAME: "Py 2.7"
+          PYTHON: "C:\\Python27-x64"
+          PYTHON_VERSION: "2.7.15"
+          PYTHON_ARCH: "64"
+          MINICONDA: C:\Miniconda-x64
+          PYQT_VERSION: "4"
+
+        # 32-bit versions, PyQT, latest to oldest.
+        
+        - NAME: "32-bit, Latest "
+          PYTHON: "C:\\Python36"
+          PYTHON_VERSION: "3.6.3"
+          PYTHON_ARCH: "32"
+          MINICONDA: C:\Miniconda36
+          PYQT_VERSION: "5"
+
+        - NAME: "32-bit, Oldest"
+          PYTHON: "C:\\Python27"
+          PYTHON_VERSION: "2.7.15"
+          PYTHON_ARCH: "32"
+          MINICONDA: C:\Miniconda
+          PYQT_VERSION: "4"
+
+        # PySide versions
+
+# Does not pass tests!  
+#
+#        - NAME: "PySide"
+#          # Wheels only available up to Python 3.4, win32.
+#          PYTHON: "C:\\Python34-x64"
+#          PYTHON_VERSION: "3.4.8"
+#          PYTHON_ARCH: "32"
+#          MINICONDA: C:\Miniconda34
+#          PYQT_VERSION: "None"
+#          QT_API: "pyside"
+#
+#        - NAME: "PySide2"
+#          PYTHON: "C:\\Python36-x64"
```

Comment:
This would be interesting to have, does appveyor has something similar to travis allowed-failure ? That way we can set it up and watch it but without it preventing to merge PRs.